### PR TITLE
OLP-1269: Setup docker compose

### PR DESCRIPTION
**Pull request by KevinOneLedger**
1. Created default docker-compose files for build and node containers
2. Added scripts for starting, updating and stopping the network (Written in bash, might need to change this for windows)

### CODE REVIEW

File: DOCKER-network/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    image: olprotocol
+    volumes:
+      - ~/oldata/docker/bin:/home/ubuntu/go/bin
+      - ~/oldata/docker/1-Node:/opt/data/devnet
+    command: >
+      bash -c "cd /opt/data/devnet &&
+              olfullnode node --root /opt/data/devnet >> /opt/data/devnet/olfullnode.log 2>&1"
+    networks:
+      vpcbr:
+        ipv4_address: 10.5.0.11
```
Comment (Jedi2002) [inline DOCKER-network/docker-compose.yml:32]: Is there a reason for not using the bridge network ?

File: DOCKER-network/stop.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# shellcheck disable=SC2046
+docker container kill $(docker ps -q)
+docker system prune -f
```
Comment (Jedi2002) [inline DOCKER-network/stop.sh:5]: I think we should remove system prune , and just remove this network instead .

Comment (kingpinXD) [inline DOCKER-network/stop.sh:5]: Can we just remove this specific network instead of using system prune

**Review by kingpinXD — APPROVED** [review]
Minor comment . Looks fine otherwise

File: DOCKER-network/stop.sh (hunk 2 above)
Comment (KevinOneLedger) [reply DOCKER-network/stop.sh:5]: Yea I can look into that, I was assuming that developers would run one network on their machine at a time.

**Pull request merged**